agency/test/services/schema.test.ts

```diff
@@ -175,60 +198,66 @@ describe('\'schema\' service', () => {
 
         const ariesAgentCreate = jest.fn();
         ariesAgentCreate
-          .mockReturnValueOnce(Promise.resolve('schema-author-transaction-id'))
-          .mockReturnValueOnce(Promise.resolve({
+          .mockResolvedValueOnce('schema-author-transaction-id')
+          .mockResolvedValueOnce({
             messages_attach: [{
               '@id': 'schema-message-attachment-id'
             }]
-          }))
-          .mockReturnValueOnce(Promise.resolve('schema-id'))
-          .mockReturnValueOnce(Promise.resolve('credential-definition-author-transaction-id'))
-          .mockReturnValueOnce(Promise.resolve({
+          })
+          .mockResolvedValueOnce('schema-id')
+          .mockResolvedValueOnce('credential-definition-author-transaction-id')
+          .mockResolvedValueOnce({
             messages_attach: [{
               '@id': 'credential-definition-message-attachment-id'
             }]
-          }))
-          .mockReturnValueOnce(Promise.resolve(undefined));
+          })
+          .mockResolvedValueOnce(undefined);
         ariesAgentService.create = ariesAgentCreate;
 
-        const params = { profile, headers: context.params.headers } as unknown as IssuerServiceParams;
         await expect(schemaService.create(schema as unknown as SchemaServiceModel, params))
           .rejects.toThrowError(CredDefError);
       });
+
+    it('should throw an error if differed schema endrosement was unsuccessful', async () => {
```

Comment:
Nit.: differed->deferred

Reply:
I could say I was coding on my phone and autocomplete kicked in but I don't think I can throw the blame there 🙃. Will edit.